backend/src/main/java/com/bakdata/conquery/models/events/stores/root/ColumnStore.java

```diff
@@ -105,7 +105,7 @@ default long estimateTypeSizeBytes() {
 	@JsonIgnore
 	<T extends ColumnStore> T createDescription();
 
-	public static <T extends ColumnStore> T emptyCopy(T store) {
+	static <T extends ColumnStore> T emptyCopy(T store) {
```

Comment:
Das ist dann `Package Private`, nicht war?

Reply:
Das sieht man hier in dem Ausschnitt nicht, aber ColumnStore ist ein public Interface. Dadurch sind die Methoden implizit public